Frontend/app/root/dashboard.html

```diff
@@ -0,0 +1,66 @@
+<div class="row tab-instruction">
+  <div class="col-xs-12 col-sm-9 col-md-8">
+    <h3>
+      Your Dashboard
+    </h3>
+    <p *ngIf="role == 'teacher' || role == 'admin'">
+      You will find all of the classes you are currently teaching or have taught in the past under the "Taught Classes" tab. If you are also enrolled in other classes, you will see those under the "Enrolled Classes" tab.
+    </p>
+    <p *ngIf="role == 'student'">
+      Here you will find all of the clases that you are currently enrolled in as a student. Click on the "View Class Detail Page" button to view more information about a specific class.
+    <br/>
+  </div>
+</div>
+
+<!-- Teacher Dashboard -->
+<div *ngIf="role == 'teacher' || role == 'admin'">
+  <ul class="nav nav-tabs" role="tablist">
+    <li role="presentation" class="active">
+      <a href="#taught" aria-controls="taught-classes" role="tab" data-toggle="tab">
+        <h4>Taught Classes</h4>
```

Comment:
Can we change this to something like "Classes where I am a Teacher"? I realized this when looking at the screen that the past tense seems confusing if I am currently teaching a class and would look rather in Enrolled Classes